### PR TITLE
Created submissions link on nav bar

### PR DESCRIPTION
**Pull request by ShashankP19**
Created submissions link on nav bar

### CODE REVIEW

File: migrations/schema.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
 -- MySQL dump 10.13  Distrib 5.7.23, for Linux (x86_64)
 --
--- Host: localhost    Database: cpjudge
+-- Host: localhost    Database: gocoder_dev
```
Comment (pvgupta24) [inline migrations/schema.sql]:
```suggestion
-- Host: localhost    Database: cpjudge
```

File: migrations/schema.sql [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 -- ------------------------------------------------------
--- Server version	5.7.23-0ubuntu0.18.04.1
+-- Server version	5.7.23-0ubuntu0.16.04.1
```
Comment (pvgupta24) [inline migrations/schema.sql]:
```suggestion
-- Server version	5.7.23-0ubuntu0.18.04.1
```
Don't change this file in this PR. We'll add to .gitignore

**Comment by pvgupta24** [comment]
From next time, please checkout a new branch from master, as it won't show previous commits already merged from master,

**Comment by ShashankP19** [comment]
I made the necessary changes in schema.sql. I'll create a new branch next time.

**Pull request merged**